db/migrate/20150307034617_create_models_based_on_eer.rb

```diff
@@ -1,8 +1,16 @@
 class CreateModelsBasedOnEer < ActiveRecord::Migration
   def change
+    create_table :roles do |t|
+      t.string :name, null: false, limit: 20
+      t.string :description, limit: 255
+      t.string :key, null: false, limit: 10
+
+      t.timestamps null: false
+    end
+
     create_table :users do |t|
       ## Database authenticatable
-      t.string :email, limit: 50,              null: false, default: ''
+      t.string :email, limit: 50, null: false, default: ''
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.